### ISSUE TITLE
Celeste: make Rinkeby testing flow easier for users

### ISSUE THREAD

**Issue by 0xGabi** [1Hive/gardens#204]
Allow Garden users to test the whole dispute flow.

**Comment by 0xGabi** [comment]
@fabriziovigevani will contact the test users to ask them if they can unstake from Celeste on Rinkeby so we can control the outcomes in future user testings.

**Comment by fabriziovigevani** [comment]
Already asked a few and we now have 12 keepers. The other addresses activated could find them in the typeform list so i guess it's most of keepers directly requesting through Discord. 
CurlyBracket made a good point when i requested him to deactivate his HNY, he pointed out what if another garden would also like to test asynchronously (that's their case for BrightDAO) and couldn't give him a better answer but to have a synchronous party like we did the other time for Celeste. Otherwise it's going to be difficult to ensure activated keepers that are online will be selected. We could also ask keepers that are testing the whole flow of gardens that when they are done, they deactivate their HNY.

**Comment by fabriziovigevani** [comment]
Already asked a few and we now have 12 keepers. The other addresses activated could find them in the typeform list so i guess it's most of keepers directly requesting through Discord.

CurlyBracket made a good point when i requested him to deactivate his HNY, he pointed out what if another garden would also like to test asynchronously (that's their case for BrightDAO) and couldn't give him a good answer. We ended agreeing that we could have a synchronous party like we did the other time for Celeste. Otherwise it's going to be difficult to ensure activated keepers that are online will be selected. We could also ask keepers that are testing the whole flow of gardens that when they are done, they deactivate their HNY.
